modules/series-service-impl/src/main/java/org/opencastproject/series/impl/persistence/SeriesServiceDatabaseImpl.java

```diff
@@ -239,29 +236,17 @@ public void deleteSeriesProperty(String seriesId, String propertyName)
    */
   @SuppressWarnings("unchecked")
   @Override
-  public Iterator<Tuple<DublinCoreCatalog, String>> getAllSeries() throws SeriesServiceDatabaseException {
+  public List<SeriesEntity> getAllSeries() throws SeriesServiceDatabaseException {
```

Comment:
How large is a SeriesEntity object?  We're loading all of them, so I can see this being a availability issue with a large enough number of series...